nvc/extract.go

```diff
@@ -0,0 +1,125 @@
+package nvc
+
+import (
+	"errors"
+	"fmt"
+	"io"
+	"os"
+	"strconv"
+)
+
+func ReadMagic(r io.Reader) error {
```

Comment:
This function doesn't need to be exported (does it?)

Reply:
:boom: